### PR TITLE
Monitors sub section creation

### PR DESCRIPTION
**Pull request by l0k0ms**
This is only for the english version of the doc.

All content in `/monitoring`, `/guides/montiors`, `/guides/outliers`, `/guides/anomalies` have been moved into the `/monitors` sub section. Content is pretty much the same, only minor updates.

This PR is up to date with : 

* Master
* Chef guide merging: https://github.com/DataDog/documentation/pull/1688
* Azure guide merging : https://github.com/DataDog/documentation/pull/1689
* Infrastructure merging: https://github.com/DataDog/documentation/pull/1717

Preview: https://d2cx6t4ssmwdrv.cloudfront.net/gus/monitors/monitors/

Preview monitor_types: https://d2cx6t4ssmwdrv.cloudfront.net/gus/monitors/monitors/monitor_types/

TO DO: 
- update in app links

### CODE REVIEW

File: content/integrations/chef.md [hunk 1]
```diff
@@ -5,23 +5,77 @@ kind: integration
...
-* Get real-time reports on Chef client runs
-* Track key Chef performance metrics across all your servers
-* Quickly identify and discuss failed Chef runs with your team
+We also provide a Chef [Execution and Report Handler](https://docs.chef.io/handlers.html) that can capture `chef-client` failures as well as metrics related to the Chef run, such as timing and resources updated.
 
 ## Setup
-### Installation
+### Deploying the Agent
+
+We created a cookbook to assist with Agent deployment.
```
Comment (irabinovitch) [inline content/integrations/chef.md]: The [datadog](https://supermarket.chef.io/cookbooks/datadog) Chef cookbook is available to automate your Datadog Agent installation and configuration.

File: content/integrations/chef.md [hunk 2]
```diff
@@ -5,23 +5,77 @@ kind: integration
...
+
+Install the latest released version of the Datadog Chef cookbook from the [Community Site](https://supermarket.chef.io/cookbooks/datadog) via knife, and upload to your Chef Server:
```
Comment (irabinovitch) [inline content/integrations/chef.md]: s/community site/Supermarket/

Comment (irabinovitch) [reply content/integrations/chef.md]: Should this link to the git repo too?

File: content/integrations/chef.md [hunk 3]
```diff
@@ -5,23 +5,77 @@ kind: integration
...
+
+    knife cookbook site install datadog
+    knife cookbook upload datadog
+
+You may also be using another tool to manage your cookbook workflow, such as [Berkshelf](http://berkshelf.com/) or [Librarian Chef](https://github.com/applicationsonline/librarian-chef), in which case you will need to add a line like this to your Berksfile/Cheffile:
+
+    cookbook 'datadog'
```
Comment (irabinovitch) [inline content/integrations/chef.md]: This is out of scope for this document. Users using Chef will know how to mange berkshelf or librarian should they wish to

File: content/integrations/chef.md [hunk 4]
```diff
@@ -5,23 +5,77 @@ kind: integration
...
+
+And follow the instructions for your tool to upload the cookbook to your Chef Server.
+
+Before adding the cookbook's recipe to your node's `run_list`, you need to add account-specific details to be provided to the Agent configuration file.
```
Comment (irabinovitch) [inline content/integrations/chef.md]: Before adding the cookbook's recipe to your node's `run_list`, you must add your Datadog account credentials such as API keys via Chef attributes.

File: content/integrations/chef.md [hunk 5]
```diff
@@ -5,23 +5,77 @@ kind: integration
...
+
+Then upload your role file to Chef Server like so:
+
+    knife role from file roles/base.rb
+
+The next time Chef runs, it should install the Agent and set the configuration file with the API and application keys.
+
+**NOTE:** If you are using another cookbook to define these attributes, use a higher attribute precedence level than `default`.
+
+### Deploying the Handler
```
Comment (irabinovitch) [inline content/integrations/chef.md]: "Report Handler"

File: content/integrations/chef.md [hunk 6]
```diff
@@ -5,23 +5,77 @@ kind: integration
...
+
+In order to further raise the visibility of your Chef runs, you may use the Datadog Chef Handler to monitor your Chef execution.
```
Comment (irabinovitch) [inline content/integrations/chef.md]: Datadog offers a Chef Report Handler which reports metrics and events from your Chef runs to Datadog.  Once installed the report handler will submit metrics on Chef run timing and resource changes. Events are also created to allow tracking of Chef run success and failure rates.

File: content/integrations/chef.md (hunk 1 above)
Comment (l0k0ms) [reply content/integrations/chef.md]: 👍

File: content/integrations/chef.md (hunk 2 above)
Comment (l0k0ms) [reply content/integrations/chef.md]: 👍

File: content/integrations/chef.md (hunk 3 above)
Comment (l0k0ms) [reply content/integrations/chef.md]: 👍

File: content/integrations/chef.md (hunk 4 above)
Comment (l0k0ms) [reply content/integrations/chef.md]: 👍

File: content/integrations/chef.md (hunk 5 above)
Comment (l0k0ms) [reply content/integrations/chef.md]: 👍

File: content/integrations/chef.md (hunk 6 above)
Comment (l0k0ms) [reply content/integrations/chef.md]: 👍
